config/sys.config

```diff
@@ -87,6 +87,14 @@
                   {formatter_config, [time, " ", message, "\n"]}
                ]}
              ]}
+           ]},
+           {epoch_sync_lager_event, [
+             {handlers, [
+               {lager_file_backend, [
+                  {file, "log/epoch_sync.log"}, {level, debug},
```

Comment:
Is this rules reroute only debug logs or all logs is lower than debug level?
I wonder if this changes affect the `lager_console_backend` handler above and docker logs?

Reply:
As I understand it, all the log done using `epoch_sync:XXX` functions end up in `epoch.sync.log`, this configuration is to filter them.

Reply:
Does not sound good, please don't merge until I test it locally